experimental/dds/tree2/src/test/class-tree/schemaFactory.examples.spec.ts

```diff
@@ -113,4 +112,10 @@ describe("Class based end to end example", () => {
 		const theTree = factory.create(new MockFluidDataStoreRuntime(), "tree");
 		setup(theTree);
 	});
+
+	it("config2", () => {
```

Comment:
A bit more description of what this is actually testing would be useful

Reply:
```suggestion
	// Confirm that the alternative syntax for the config from the example above (config2) actually works.
	it("config2", () => {
```